packages/netlify-cms-core/src/reducers/entryDraft.js

```diff
@@ -22,10 +24,11 @@ const initialState = Map({
   mediaFiles: List(),
   fieldsMetaData: Map(),
   fieldsErrors: Map(),
+  fieldValidators: Map(),
   hasChanged: false,
 });
 
-const entryDraftReducer = (state = Map(), action) => {
+const entryDraftReducer = (state = initialState, action) => {
```

Comment:
I don't know why we weren't doing this already, but if it causes bugs, we should just fix them. `intialState` becomes the default from the first time you open the editor, even if you go to other screens after, so we should just use it from the start.